wordpress-mobile/WordPress-Aztec-iOS
# Specify the encoding to the HTML parser

I discovered an issue with the converter not handling non-ASCII characters properly.

To test:
- Change `content.html` to have some non-ASCII text
- Launch the editor demo and verify it shows correctly

Needs Review: @diegoreymendez
